OpenSim/Common/C3DFileAdapter.h

```diff
@@ -45,14 +51,23 @@ class OSIMCOMMON_API C3DFileAdapter : public FileAdapter {
     ~C3DFileAdapter()                                = default;
 
     C3DFileAdapter* clone() const override;
+
+    void setLocationForForceExpression(const ForceLocation location) {
+        _location = location;
+    }
+
+    const ForceLocation getLocationForForceExpression() const {
+        return _location;
+    }
     
     /** Read in a C3D file into separate markers and forces tables of type
         TimeSeriesTableVec3. The markers table has each column labeled by its
         corresponding marker name. For the forces table, the data are grouped
         by sensor (force-plate #) in force, point and moment order, with the
         respective *f#*, *p#* and *m#* column labels. */
     static
-    Tables read(const std::string& fileName);
+    Tables read(const std::string& fileName, 
```

Comment:
Perhaps add a comment to the doxygen indicating the scenarios in which one might want to use each force location?